src/Areas/Canvases/CanvasesSetup.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using AzureMcp.Areas.Canvases.Commands.Workbook;
+using AzureMcp.Areas.Canvases.Services;
+using AzureMcp.Commands;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace AzureMcp.Areas.Canvases;
+
+public class CanvasesSetup : IAreaSetup
```

Comment:
Can you explain why you are using Canvases?  That doesn't ring a bell for me.  Why not Workbooks or something like that.